<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QMainWindow" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KARRA NAMI ACCOUNT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #5A67D8, stop:1 #2D3748);
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <spacer name="topSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="registrationCard">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        background-color: rgba(255, 255, 255, 0.98);
        border: 1px solid rgba(0, 0, 0, 0.08);
        border-radius: 12px;
        padding: 25px;
        box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
       </string>
      </property>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Create Your Account</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string notr="true">
           font-size: 26px;
           font-weight: 700;
           color: #1A202C;
           padding-bottom: 20px;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
          </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="companyNameInput">
         <property name="placeholderText">
          <string>Company Name</string>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="regNoInput">
         <property name="placeholderText">
          <string>Registration Number</string>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailInput">
         <property name="placeholderText">
          <string>Email Address</string>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mobileInput">
         <property name="placeholderText">
          <string>Mobile Number</string>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordInput">
         <property name="placeholderText">
          <string>Password</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirmPasswordInput">
         <property name="placeholderText">
          <string>Confirm Password</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
           padding: 12px 15px;
           border: 2px solid #E2E8F0;
           border-radius: 8px;
           font-size: 15px;
           background-color: #F8FAFC;
           font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
           color: #2D3748;
           transition: border-color 0.2s;
          </string>
         </property>
         <property name="minimumHeight">
          <number>48</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="registerButton">
         <property name="text">
          <string>Create Account</string>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QPushButton {
               background-color: #3182CE;
               width:50px;
               color: white;
               padding: 14px;
               border: none;
               border-radius: 8px;
               font-size: 16px;
               font-weight: 600;
               font-family: 'Inter', 'Segoe UI', Arial, sans-serif;
               transition: background-color 0.2s;
           }
           QPushButton:hover {
               background-color: #2B6CB0;
           }
           QPushButton:pressed {
               background-color: #2C5282;
               padding: 15px 14px 13px 14px;
           }
           QPushButton:disabled {
               background-color: #A0AEC0;
           }
          </string>
         </property>
         <property name="minimumHeight">
          <number>50</number>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>